adagrams/game.py

```diff
@@ -1,11 +1,79 @@
+import random
+from collections import Counter
+LETTER_POOL20 = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
 def draw_letters():
-    pass
+    values = list(LETTER_POOL20.values())
+    keys = list(LETTER_POOL20.keys())
+    players_hand =random.sample(keys, counts = values, k =10)
+    return players_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
-
+    word = word.upper()
+    available_bank = Counter(letter_bank)
+    for letter in word:
+        if letter in letter_bank and available_bank[letter]:
```

Comment:
Note that available_bank[letter] returns an integer representing the number of times a letter is in available_bank or 0 if it's not in available_bank. 0 is Falsy so this check does achieve what you're trying to do. 

However, you could express the intent of this line of code more clearly if you wrote:
```python
if letter in letter_bank and letter in available_bank:
```